docs/readme_de.md

```diff
@@ -157,10 +157,15 @@ Du kannst mehrere, mit Kommas separierte, Werte in der bg_color Option angeben,
 
 #### Exklusive Optionen der Sprachen-Karte:
 
-- `hide` - Verbirgt die angegebenen Sprachen von der Karte _(Komma separierte Werte)_
-- `hide_title` - _(Boolean)_
-- `layout` - Wechsel zwischen den zwei verfügbaren Layouts `default` & `compact`
-- `card_width` - Lege die Breite der Karte manuell fest _(Zahl)_
+- `hide` - Ausblenden der angegebenen Sprachen auf der Karte _(Comma-separated values)_. Standard: `[] (leeres Array)`.
+- `hide_title` - _(boolesch)_. Voreinstellung: „false“.
+- `layout` - Umschalten zwischen zwei verfügbaren Layouts `default` & `compact`. Standard: "default".
+- `card_width` - Stellen Sie die Breite der Karte manuell ein _(number)_. Standard '300'.
+- `langs_count` - Zeige mehr Sprachen auf der Karte, zwischen 1-10 _(number)_. Standard `5`.
+- `exclude_repo` - Angegebene Repositories ausschließen _(Comma-separated values)_. Standard: `[] (leeres Array)`.
+- `custom_title` - Legt einen benutzerdefinierten Titel für die Karte fest _(string)_. Standard `Am häufigsten verwendete Sprachen`.
+- `disable_animations` - Deaktiviert alle Animationen in der Karte _(boolean)_. Voreinstellung: „false“.
+- `hide_progress` - Es verwendet die kompakte Layoutoption, blendet Prozentsätze aus und gleicht die Balken an. Standard `false`.
```

Comment:
Again, same thing. Please only update the English readme.

Reply:
> Hi, thanks for the high quality PR! A few things before we merge:
Hello, and thank you.

> * No need to make changes to the docs in all languages! It's really amazing that you've taken the time to do that, but we're unfortunately in the process of migrating to an automated translation system, so we're not accepting any PRs regarding translated docs. I appreciate that you've taken the time to do that, and I apologize for the miscommunication.
Oh, okay, the translation system seems incredible.

> * I would think this is a valuable PR, but I'm not sure about the use case. Why would someone want to make the progress bars equal? It would be misleading to think that they spent equal time on all their languages. Hiding the percentages is a good feature, but again, I'm not sure about switching to compact mode. If the equal bars feature is removed, hiding percentages would make as much sense in the default layout as in the compact layout. Please consider my suggestion! @rickstaa please feel free to add your thoughts...

About the progress bar. In the `hire_progress=true` scenario, I only thought of it as juice, having the language colors instead of removing the bars. You're right; I can hide it.

About the default mode, I thought they would be so much alike, also, having a language per line without a progress bar looked a bit unnecessarily space-consuming.

I agree with anything you guys decide and will fix the issues.